client.rb

```diff
@@ -3,6 +3,7 @@
 require 'net/http'
 require 'uri'
 require 'yaml'
+require 'erb'
```

Comment:
This should be moved to the top as the other `require`s seem to be sorted alphabetically.

Reply:
Done in 6143af1